nifi-nar-bundles/nifi-graph-bundle/nifi-other-graph-services/src/main/java/org/apache/nifi/graph/TinkerpopClientService.java

```diff
@@ -0,0 +1,538 @@
+            .build();
+
+    public static final PropertyDescriptor USER_NAME = new PropertyDescriptor.Builder()
+            .name("user-name")
+            .displayName("Username")
+            .description("The username used to authenticate with the gremlin server." +
+                    " Note: when using a remote.yaml file, this username (if set) will overload any username set in the yaml file.")
+            .required(false)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
+            .build();
+
+    public static final PropertyDescriptor PASSWORD = new PropertyDescriptor.Builder()
+            .name("password")
+            .displayName("Password")
+            .description("The password used to authenticate with the gremlin server." +
+                    " Note: when using a remote.yaml file, this password (if set) will override any password set in the yaml file")
+            .required(false)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
+            .sensitive(true)
+            .build();
+
+    public static final PropertyDescriptor EXTRA_RESOURCE = new PropertyDescriptor.Builder()
+            .name("extension")
+            .displayName("Extension JARs")
+            .description("A comma-separated list of Java JAR files to be loaded. This has no practical effect unless combined " +
+                    "with the Extension Classes property which provides a list of particular classes to use as extensions for the " +
+                    "parsing engine.")
+            .dependsOn(CONNECTION_SETTINGS, YAML_SETTINGS)
+            .defaultValue(null)
+            .identifiesExternalResource(ResourceCardinality.MULTIPLE, ResourceType.FILE, ResourceType.DIRECTORY, ResourceType.URL)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
+            .dynamicallyModifiesClasspath(true)
+            .build();
+
+    public static final PropertyDescriptor EXTENSION_CLASSES = new PropertyDescriptor.Builder()
+            .name("extension-classes")
+            .displayName("Extension Classes")
+            .addValidator(Validator.VALID)
+            .description("A comma-separated list of fully qualified Java class names that correspond to classes to implement. This " +
+                    "is useful for services such as JanusGraph that need specific serialization classes. " +
+                    "This configuration property has no effect unless a value for the Extension JAR field is " +
+                    "also provided.")
+            .dependsOn(EXTRA_RESOURCE)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
+            .dependsOn(CONNECTION_SETTINGS, YAML_SETTINGS)
+            .required(false)
+            .build();
+
+    public static final PropertyDescriptor SSL_CONTEXT_SERVICE = new PropertyDescriptor.Builder()
+            .name("tinkerpop-ssl-context-service")
```

Comment:
Recommend removing the prefix for the property name.
```suggestion
            .name("ssl-context-service")
```

Reply:
Removed.